### DISCUSSION TITLE
为何邮件通知在首次发表时没有通知？

### DISCUSSION THREAD

**Discussion by elkan1788** [walinejs/waline#544] [Q&A] [answered]
你好，参考了官方的配置手册，配置了邮箱通知的相关如下参数：

![image](https://user-images.githubusercontent.com/1217096/132985903-3239954c-4f53-476e-9cb4-814059ead5cd.png)

![image](https://user-images.githubusercontent.com/1217096/132986041-226d4f75-92ac-43fa-977d-f836c3978445.png)

 结果发现首次评论时，管理员邮箱是不能收到通知，只有后续的交流中才可以收到，这是为何？

另外看网友有配置过管理员的邮件模板，尝试在`index.js`中添加也是无法生效，仍然首次评论不能收到通知。
![image](https://user-images.githubusercontent.com/1217096/132985965-82a30e87-9925-438a-ab75-739fe3b04a61.png)

请问能否帮忙分析下是什么原因，谢谢。

**Comment by lizheming** [comment]
如果配了其它第三方通知的话，是不会发送新评论邮件通知的。具体的逻辑可以看这里 https://github.com/walinejs/waline/blob/main/packages/server/src/service/notify.js#L234-L246

**Reply by elkan1788** [reply]
> 如果配了其它第三方通知的话，是不会发送新评论邮件通知的。具体的逻辑可以看这里 https://github.com/walinejs/waline/blob/main/packages/server/src/service/notify.js#L234-L246

@lizheming 你好，这里确认是没有设置其他通知，也未配置 `AUTHOR_EMAIL` 参数，但不知道为何就是首次评论收不到邮件通知，后续的互动是有邮件通知的。

**Reply by Masle1** [reply]
你好，请问现在找到解决方案了吗

**Comment by cuihuaerme** [comment, marked as answer]
是不是漏了添加 AUTHOR_EMAIL，我加上再部署一下就好了

**Reply by elkan1788** [reply]
升级到最新版本是可以正常收到回复，谢谢。
